apps/console/src/features/users/components/add-user.tsx

```diff
@@ -314,6 +339,16 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
 
                                     let passwordRegex: string = "";
 
+                                    if (passwordConfig) {
```

Comment:
Instead of nested if s , let's combine two conditions using &&

Reply:
Resolved in [258cafe](https://github.com/wso2/identity-apps/pull/4652#:~:text=address%20comments-,258cafe,-Merge%20state)